### PR TITLE
Calculator testing completion

### PR DESCRIPTION
**Pull request by RNtaate**
## What does this PR do?
- [X] Installs react-testing library, jest-dom, react-test-renderer testing dependencies.
- [X] Creates a __test__ folder in the components folder to hold all test files for the components.
- [X] Creates tests for various components using jest and React-testing-library such as App, Button, Home, Quote, ButtonPanel, e.t.c
- [X] Performs snapshot testing for various components with jest.
- [X] Creates a __test__ folder in the logics folder and adds the calculate test file to it.
- [X] Adds unit tests for the calculate.js file.
- [X] Fixes eslint linter offenses.

### CODE REVIEW

File: src/logic/__test__/calculate.test.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  expect(newData.operation).toBe('+');
+
+  myData.total = '3';
+  myData.operation = '+';
+  myData.next = '2';
+  newData = calculate('=', myData);
+  expect(newData.total).toBe('5');
+  expect(newData.next).toBeNull();
+  expect(newData.operation).toBeNull();
+});
```
Comment (Torres-ssf) [inline src/logic/__test__/calculate.test.js]: - [ ] We need to add a unit test for each math operator `( +, -, *, /, %)`. Example: the test should ensure that `5 * 10 = 50`.

- [ ] We also need to add a unit test to ensure that the buttons like `(+/-, AC, =, ., )` are working as expected. Example: `+/-` should turn `-4` into `4`

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes Required :muscle: 
Hi @RNtaate :hand: 
Great job testing your application :clap: 
I can see that a lot of tests were implemented. That is really nice.
But we need to more unit tests to our calculate.js. The reason for this is to make sure that all operations from calculate.js are working properly.
Screenshot from 2021-04-12 11-26-20


- [ ] Check my comment below

Happy coding :computer: 
**Sérgio Torres**

**Comment by RNtaate** [comment]
## Changes made
- [X] Added unit test for the various math operators as requested by the reviewer.
- [X] Added unit tests for buttons like `AC`, `+/-`, `.`

**Review by Torres-ssf — APPROVED** [review]
### PR Approved :+1: :clap: 
Hello again, @RNtaate :hand: 
Nice job adding more unit tests 
You may merge the PR :+1: 
Good luck with the next project :smile: 

Happy coding :computer: 
**Sérgio Torres**

**Pull request merged**